articles/using-select-inputs-to-edit-your-data.mdx

```diff
@@ -114,14 +120,18 @@ _inputs:
 </comp.MultiCodeBlock>
 
 ## Multichoice
+
 Segmented controls input for selecting multiple values from a set of options. Functionally equivalent to a [Multiselect](#multiselect) input.
 
 Multichoice inputs are shown for inputs configured with the type `multichoice`.
 
+<comp.DocsImage path="/documentation/static/inputs/select/multichoice.png" alt=""/>
```

Comment:
Consider using multichoice-with-icons.png to show the UI with nice icons?

Reply:
Yeah I considered it when uploading, but then we would have to change the corresponding codeblock as well. Doing so would really add a lot of complexity, and I think it's better to show a really simple example. I was tempted to write an extra example showing off the multi with icons, think it's worth it?

Reply:
>  we would have to change the corresponding codeblock as well. Doing so would really add a lot of complexity

Oh yep, great point. I'm content with how it is, up to you! I've approved the PR anyway.